boards/x86/acrn/board.h

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Remove this file, its not needed anymore.

Reply:
done